mlocal/checks/version.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	"go/build"
+	"os"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Printf("E: missing target Go version\n")
```

Comment:
Should this print to stderr?

Reply:
I don't think it makes a difference, given how this code is used specifically to test the version of the compiler, but yes, that can be changed.